pkg/defaults/policies/files/severity_high_privileged.json

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "b1df1abb-e5a5-4ff7-98fe-1d28a22b55d8",
+  "name": "Severity Important and Privileged",
```

Comment:
Severity exists on policy, vulns, etc. Make the policy title clearer. `Privileged Container(s) with Important and Critical CVE(s)`